CMakeLists.txt

```diff
@@ -1,11 +1,18 @@
 project(EUDAQ)
 set(${PROJECT_NAME}_VERSION_MAJOR 2 )
-set(${PROJECT_NAME}_VERSION_MINOR 0 )
+set(${PROJECT_NAME}_VERSION_MINOR 4 )
```

Comment:
Version 2.3.0 currently, right?

Reply:
Yes, I want to tag v2.4.0 immediately after merge because that will be a change how to interface EUDAQ

Reply:
Okay.